README.md

```diff
@@ -135,11 +135,11 @@ Here are details of different MS-AMP optimization levels:
 
 ### Accuracy: no loss of accuracy
 
-We evaluated the training loss and validation performance of a typical model, Swin-Transformer, using both MS-AMP O2 and FP16 AMP. Our observations showed that the model trained with MS-AMP O2 mode achieved comparable performance to those trained using FP16 AMP. This demonstrates the effectiveness of the Mixed FP8 O2 mode in MS-AMP.
+We evaluated the training loss and validation performance of three typical models, Swin-Transformer, DeiT and RoBERTa, using both MS-AMP O2 and FP16 AMP. Our observations showed that the models trained with MS-AMP O2 mode achieved comparable performance to those trained using FP16 AMP. This demonstrates the effectiveness of the Mixed FP8 O2 mode in MS-AMP.
 
-Here is the result for Swin-T model:
+Here is the result for Swin-T model and DeiT-S:
```

Comment:
`Here is the result`=>`Here are the results`

Reply:
Fixed.